massa-pool-exports/src/controller_traits.rs

```diff
@@ -39,8 +42,8 @@ pub trait PoolController: Send + Sync {
     /// Check if the pool contains a list of operations. Returns one boolean per item.
     fn contains_operations(&self, operations: &[OperationId]) -> Vec<bool>;
 
-    /// Add denunciations to pool. Simply print a warning on failure.
-    fn add_denunciation(&mut self, denunciation: Denunciation);
+    /// Get the number of denunciations in the pool
+    fn get_denunciations(&self) -> Vec<Denunciation>;
```

Comment:
this gets a full copy of all denunciations (potentially very heavy) instead of the number of denunciations as stated in the docs

Reply:
removed